src/components/auth/register/helpers/avatarsData.tsx

```diff
@@ -0,0 +1,15 @@
+export const assignRandomProfilePicture = () => {
+  const avatars = [
+    '/avatars/avatar-frog-blue.svg',
+    '/avatars/avatar-frog-yellow.svg',
+    '/avatars/avatar-lion-blue.svg',
+    '/avatars/avatar-lion-yellow.svg',
+    '/avatars/avatar-panda-yellow.svg',
+    '/avatars/avatar-sheep-blue.svg',
+    '/avatars/avatar-sheep-yellow.svg',
+    '/avatars/avatar-squirrel-blue.svg',
+    '/avatars/avatar-squirrel-blue.svg',
```

Comment:
as a best practice, the array should contain only the names, but not the path and the file extension, so that path and file to be configurable separately and changeable in future.